### PR TITLE
Issue 1: Draft of Introduction to Testing in W3C

### PR DESCRIPTION
**Pull request by hubenjamin**

### CODE REVIEW

File: Issue-1-Introduction.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+layout: post
```
Comment (tobie) [inline Issue-1-Introduction.md:2]: please you default for layout

File: Issue-1-Introduction.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Secondly, the web platform has dramatically increased its complexity as it has shifted from a document exchange platform to a rich application environment that enables a wide variety of web applications. It also requires a concerted testing effort in order to achieve interoperability across different implementations of web platform.
+
+Thirdly, as the scope and ecosystem of the web platform has expanded, so does the number and diversity of its stakeholders (mobile, automotive, TV, developers, etc.), which have increasingly pressing demands to push testing further into areas which bring them more value.
+
+Therefore there is a need to organize the testing effort to understand, prioritize, and find resources to meet those different requirements.
+
+W3C has organized Test The Web Platform program to serve the need of increasing testing effort on a broad spectrum of W3C specifications.
+
+W3C�s testing philosophy includes Automatic Testing, Manual Testing and Referencing Testing.
```
Comment (tobie) [inline Issue-1-Introduction.md:18]: You might have some Windows related encoding issues here. The apostrophes are all weird.

File: Issue-1-Introduction.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+Automatic Testing is always the preferred way for the majority of test assertions of W3C specifications. W3C�s Test Framework includes the combination of a Test Runner that can run testharness.js tests and a repository of test suites, and produces test reports.
+Under certain circumstances, Manual Testing is inevitable for many reasons, for example:
```
Comment (tobie) [inline Issue-1-Introduction.md:21]: Let's completely avoid the whole manual testing thing for now. We'll approach it through WebDriver testing doen the line.

File: Issue-1-Introduction.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+* Sometimes, user agent technology protects its users by specifically preventing something from being automated - for example, printing, resizing windows
+* Sometimes, you are testing offline behavior, so you may need to be temporarily disconnected from the testharness, etc.
+A Manual Test Runner can help running manual tests and collect the test results.
```
Comment (tobie) [inline Issue-1-Introduction.md:25]: See above.

File: Issue-1-Introduction.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+Secondly, the web platform has dramatically increased its complexity as it has shifted from a document exchange platform to a rich application environment that enables a wide variety of web applications. It also requires a concerted testing effort in order to achieve interoperability across different implementations of web platform.
+
+Thirdly, as the scope and ecosystem of the web platform has expanded, so does the number and diversity of its stakeholders (mobile, automotive, TV, developers, etc.), which have increasingly pressing demands to push testing further into areas which bring them more value.
+
+Therefore there is a need to organize the testing effort to understand, prioritize, and find resources to meet those different requirements.
+
+W3C has organized Test The Web Platform program to serve the need of increasing testing effort on a broad spectrum of W3C specifications.
```
Comment (tobie) [inline Issue-1-Introduction.md:16]: The above is too generic. This is meant to be an entry point to the docs, not to the whole effort. So the _how_ of testing, not the _why_.

Let's also try to stray away from corporate lingo like "Test The Web Platform program," this is supposed to be fun and get people excited to contribute.

**Comment by tobie** [comment]
You might want to hold off the intro stuff until we have a better idea of the what the whole doc site looks like.

**Comment by hubenjamin** [comment]
Ok, agree to hold it off until the whole doc site finishes

**Pull request closed without merging**